etl/assets.py

```diff
@@ -73,20 +70,24 @@ def documents_of_wikipedia_articles_with_summaries(
 
     return DocumentTuple.from_records(
         records=wikipedia_articles_with_summaries.records,
-        record_type=openai_pipeline_config.record_type,
         enrichment_type=openai_pipeline_config.enrichment_type,
     )
 
 
 @asset
-def wikipedia_articles_embeddings(
+def wikipedia_articles_embedding_store(
     documents_of_wikipedia_articles_with_summaries: DocumentTuple,
     openai_settings: OpenAiSettings,
     output_config: OutputConfig,
-) -> VectorStore:
-    """Materialize an asset of Wikipedia articles embeddings."""
+) -> None:
+    """Materialize an asset of Wikipedia articles embeddings and write it to disk."""
 
-    return OpenAiEmbeddingPipeline(
+    OpenAiEmbeddingPipeline(
         openai_settings=openai_settings,
         output_config=output_config,
-    ).create_embedding_store(documents_of_wikipedia_articles_with_summaries.documents)
+    ).create_embedding_store(
+        documents_of_wikipedia_articles_with_summaries.documents
+    ).save_local(
```

Comment:
Drop `.save_local` and put the cached embeddings (= one JSON file per document, created by LangChain's CacheBackedEmbeddings) into the "data" repo.